tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowsersecuritychange_event.html

```diff
@@ -0,0 +1,46 @@
+<head>
+  <title>mozbrowsersecuritychange</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+  <script>
+
+    async_test(function(t) {
+
+      const HTTP_URL = "http://web-platform.test:8000";
+      const HTTPS_URL = "https://web-platform.test:8443";
+
+      var urls = [ HTTP_URL, HTTPS_URL, HTTP_URL ];
+      var url_index = 0;
+
+      var expectedEvents = [
+        "insecure",
+        "secure",
+        "insecure",
```

Comment:
Let's load each page twice in a row to ensure that we don't get events when the state doesn't change. On the other hand, if that's not something we care about, the text at https://developer.mozilla.org/en-US/docs/Web/Events/mozbrowsersecuritychange should be updated to be more precise.